content/doc/developer/plugin-development/incrementals.adoc

```diff
@@ -0,0 +1,108 @@
+---
+layout: developersection
+title: "Developing plugins and Jenkins core in parallel"
+tags:
+- jenkins
+- incrementals
+- core
+- plugin
+description: "This article explains how to use Incrementals for developing plugins in parallel with Jenkins core"
+---
+
+A common problem one might stumble upon is developing an API in Jenkins core and simultaneously
+building a reference implementation inside a plugin.
+What to do, one might ask.
+
+Worry not, Incrementals are here to the rescue!
+
+With incrementals, we can create Jenkins core incremental versions which can be used along with the plugin to build the
+plugin against the very latest changes you have made to core.
+Also, it allows keeping track and switching between incremental versions.
+Not only that, it allows others to also not have to build Jenkins locally to try out the changes or collaborate with
+you on your awesome project!
+
+Sounds cool? Let's see how to go about it.
+
+== 1. Create a Pull Request in Jenkins core
```

Comment:
Headings usually are not numbered.  We rely on the table of contents and linear reading rather than numbering inside the heading
```suggestion
== Create a Pull Request in Jenkins core
```